auction-config.json

```
@@ -4,12 +4,14 @@
   "tvlInfoApiBaseUrl": "https://api.marinade.finance",
   "blacklistApiBaseUrl": "https://raw.githubusercontent.com/marinade-finance/delegation-strategy-2/master",
   "snapshotsApiBaseUrl": "https://snapshots-api.marinade.finance",
+  "scoringApiBaseUrl":  'https://scoring.marinade.finance',
```

```suggestion
  "scoringApiBaseUrl":  "https://scoring.marinade.finance",
```
The `'`s are invalid in JSON, it needs to use double quotes.
(Sorry to confuse you with the specific string in my previous comment, I took it from a TS file where it's ok.)